core/src/main/java/kafka/server/builders/KafkaApisBuilder.java

```diff
@@ -178,6 +179,7 @@ public KafkaApis build() {
                              metadataSupport,
                              replicaManager,
                              groupCoordinator,
+                             new GroupCoordinatorAdapter(groupCoordinator, time),
```

Comment:
Is there a reason for not having a setter for the GroupCoordinatorAdapter similar to `setGroupCoordinator`?

Reply:
GroupCoordinatorAdapter is an internal change so we don't have to expose it in the builder. It only wraps the GroupCoordinator passed.